src/panels/KaitoPanel.ts

```diff
@@ -241,3 +247,10 @@ function isInvalidTemplateDeploymentError(ex: unknown): ex is InvalidTemplateDep
 function isRestError(ex: unknown): ex is RestError {
     return typeof ex === "object" && ex !== null && ex.constructor.name === "RestError";
 }
+
+function listKaitoSupportedModels(): ModelDetails[] {
+    return kaitoSupporterModel.modelDetails.map((model) => ({
```

Comment:
Having it as object so that we can easily transition to next screen which will be list of models of small cards.